### PR TITLE
add Module.java class and ModuleTest.java

### LINKED ISSUE #46: Create Module class
Create the following:
- [x] Module.java
- [x] ModuleTest.java
  - [x] Yet to update inaccurate comments

### PR DESCRIPTION
**Pull request by whitesnowx**

### CODE REVIEW

**Review by tsulim — APPROVED** [review]
Looks good to me!

File: src/test/java/staffconnect/model/person/ModuleTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        assertFalse(Module.isValidModule("2103T")); // missing prefix
+        assertFalse(Module.isValidModule("C2103T")); // only 1 prefix
+        assertFalse(Module.isValidModule("CS2103TTTT")); // too many suffix
+        assertFalse(Module.isValidModule("CSCSCS2103T")); // too many prefix
+        assertFalse(Module.isValidModule("CS210310101010T")); // too many numbers
+        assertFalse(Module.isValidModule("CS21T")); // too little numbers
+        assertFalse(Module.isValidModule("cs2103t")); // suffix not capitalised
+        assertFalse(Module.isValidModule("cs2103T")); // prefix not capitalised
+
+        // valid phone numbers
```
Comment (Pluiexo) [inline src/test/java/staffconnect/model/person/ModuleTest.java:41]: Nits misleading phone numbers comments in this file, please fix in future commits!

**Review by Pluiexo — COMMENTED** [review]
Nits misleading comments in the file

File: src/test/java/staffconnect/model/person/ModuleTest.java (hunk 1 above)
Comment (whitesnowx) [reply src/test/java/staffconnect/model/person/ModuleTest.java:41]: will do

**Pull request merged**